yarn.lock

```diff
@@ -3569,6 +3601,16 @@
     redux-thunk "^2.4.1"
     reselect "^4.1.5"
 
+"@reduxjs/toolkit@^1.8.3":
```

Comment:
where did this came from? we should already have redux-toolkit in lockfile

Reply:
https://github.com/trezor/trezor-suite/pull/5751/commits/1787f128fcb07d616291d1027d9ada20efdc68a6